Kentico/kontent-starter-corporate-next-js
# Integrate Next.js preview and Kontent preview

### Motivation

Fixes #3

### Checklist


### How to test

Test that turning on preview and back off works.
Try to create a new page/post and see if it is only accessible in preview.
Check `/preview/*` routes are not accessible without `KONTENT_PREVIEW_SECRET` key.
